homeassistant/components/kraken/config_flow.py

```diff
@@ -1,5 +1,6 @@
 """Config flow for kraken integration."""
 import logging
+from typing import Any, Dict, Optional
```

Comment:
Add `from __future__ import annotations` and you shouldn't need to swap order of classes. Basically you can add it in every file.

You can read more in this [PEP](https://www.python.org/dev/peps/pep-0563/).